### ISSUE TITLE
Queue table headers not being read by screen reader

### ISSUE THREAD

**Issue by alisan16** [department-of-veterans-affairs/caseflow#12094]
Table header cells with filtering capabilities are read as "filter by column_name button" with screen readers. Column_names should be read without the "filter by" button being selected.

Suggested fix: Add <th> tags to table header

From Section 508 Audit: Data in columns with filtering capabilities are read as filter by when reviewing information with assistive technology programs and table commands. Elements should only read as filter by in the first row when the filter button has focus visually. This occurs with numerous tables throughout the web hosted application. (New: 08/21/2019) Every table cell with a header filter reads as if you can filter from that location

**Comment by hschallhorn** [comment]
@alisan16 am I correct in assuming that the acceptance criteria on this ticket is that
- [ ] The user can tab through each of the table headers
- [ ] The table headers are read by the screen reader as just the column name (without the "filter by", unless the filter button is selected)

**Comment by alisan16** [comment]
@hschallhorn Yes, and that the "filter by" buttons can also be accessed by the user through tabbing and are read as "filter by column_name"

**Comment by yoomlam** [comment]
Ok, good. That is the behavior after these commits.

**Comment by yoomlam** [comment]
Here's the behavior I observe:

When testing on the master branch (I just pulled it), VO reads col name. I move right, VO reads col name (again). I move right, VO reads “Filter by <COL>“, skipping the “sort by” element group.

After my changes, VO reads the col name. When I move to the right, VO reads “Sort by <COL>“. When I move the right again, VO reads “Filter by <COL>“. This seems reasonable to me.

**Comment by araposo-tistatech** [comment]
Per the recent audit, this issue is not resolved: 

Data in columns with filtering capabilities are read as filter by when reviewing information with assistive technology programs and table commands. Elements should only read as filter by in the first row when the filter button has focus visually. This occurs with numerous tables throughout the web hosted application. (Not Resolved: 12/13/2019) Every table cell with a header filter reads as if you can filter from that location (3/26/2020 Not Resolved)
